compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmPackageGen.java

```diff
@@ -194,6 +194,7 @@ private static void addBuiltinImports(PackageID currentModule, Set<PackageID> de
         }
         dependentModuleArray.add(PackageID.INTERNAL);
         dependentModuleArray.add(PackageID.ARRAY);
+        dependentModuleArray.add(PackageID.REGEXP);
```

Comment:
Since the regexp module has dependencies on modules `lang.internal`, `lang.array`, `lang.object`, `lang.jballerina.java`, and `lang.value`, Shall we move this line after all these modules? 
Adding it just before the line
```java
dependentModuleArray.add(PackageID.STRING);
```
will be correct.